### PR TITLE
feat: controller wire up script

### PR DESCRIPTION
**Pull request by sajanrajdev**
This PR closes issue #72 

- Added generic method to wire up a vault and strategy to a certain controller to the Badger class.
- Added CLI callable function to run the new method from devMulti.
- Added the remDIGG strat, vault and recovered controller to the address book for reference.

To wire up remDIGG call:
```
brownie run scripts/badger/wire_up_controller.py wire_up_controller 0x3F61344BA56df00dad9bBcA05d98CA2AeC43Ba0B 0x99F39D495C6A5237f43602f3Ab5F49786E46c9B0 0x4055D395361E73530D43c9D4F18b0668fe4B5b91
```

### CODE REVIEW

**Comment by sajanrajdev** [comment]
Tx and events trace for remDIGG wire up:
```
Running 'scripts/badger/wire_up_controller.py::wire_up_controller'...
/home/sajan/badger/badger-multisig/venv/lib/python3.9/site-packages/brownie/network/contract.py:708: BrownieEnvironmentWarning: 'ISettV4h' defines a 'balance' function, 'ISettV4h.balance' is available as ISettV4h.wei_balance
  warnings.warn(
Same want for vault and strategy: 0x798D1bE841a82a273720CE31c822C61a67a601C3

Wiring vault: 0x8B2a18b6400338272FDD1B991F5163E21723AF60, on controller: 0x3F61344BA56df00dad9bBcA05d98CA2AeC43Ba0B...
Transaction sent: 0x88d83a8be018af445f4edddf699cdb15d4f496e40dc375d4c100df39a1a2b4df
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 1
  IController.setVault confirmed   Block: 14120743   Gas used: 47587 (0.40%)

Wiring strategy: 0x4055D395361E73530D43c9D4F18b0668fe4B5b91, on controller: 0x3F61344BA56df00dad9bBcA05d98CA2AeC43Ba0B...
Transaction sent: 0x64d820b9281da969ac27ddd0b998ed7d6ba3585c39053c81ba9d2fa6728fb99b
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 2
  IController.approveStrategy confirmed   Block: 14120744   Gas used: 46705 (0.39%)

Transaction sent: 0xdeab240a2365ada0914ee736f82f4917aaaa44062ca3d63649e12ec74a7451fa
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 3
  IController.setStrategy confirmed   Block: 14120745   Gas used: 48614 (0.41%)

Transaction sent: 0x165ce15f3ebc60ee7ef042796f01c4a33ea23d620db35b105c766c0c31dfa90f
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 62
  Gnosis Safe.approveHash confirmed   Block: 14120743   Gas used: 46206 (0.39%)

Transaction sent: 0x2ba75c33187fbef187fe010f2bedb6e60617c3282509a2ffa7b39f236298523c
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 180
  Gnosis Safe.approveHash confirmed   Block: 14120744   Gas used: 46206 (0.39%)

Transaction sent: 0xdfc707c4ac700143f9dfcb8e45e42501df61940280b1f95357d496b66309e6fb
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 0
  Gnosis Safe.approveHash confirmed   Block: 14120745   Gas used: 46206 (0.39%)

Transaction sent: 0x1671ecc5982bf6fe306ae99cc585a2faa3880c36daea8f8957dec3485999aa57
  Gas price: 0.0 gwei   Gas limit: 152296   Nonce: 63
  Gnosis Safe.execTransaction confirmed   Block: 14120746   Gas used: 112999 (74.20%)

Transaction was Mined
---------------------
Tx Hash: 0x1671ecc5982bf6fe306ae99cc585a2faa3880c36daea8f8957dec3485999aa57
From: 0x15b8Fe651C268cfb5b519cC7E98bd45C162313C2
To: 0xB65cef03b9B89f99517643226d76e286ee999e77
Value: 0
Function: Gnosis Safe.execTransaction
Block: 14120746
Gas Used: 112999 / 152296 (74.2%)

Events In This Transaction
--------------------------
└── Gnosis Safe (0xB65cef03b9B89f99517643226d76e286ee999e77)
    └── ExecutionSuccess
        ├── txHash: 0x5835611161d09495ee5850ebbfb43aacc0b117d988688859d7acf99c54eef98a
        └── payment: 0

```

**Review by tabshaikh — APPROVED** [review]
LGTM wires up controller to setVault and strategy
Screenshot 2022-02-01 at 8 39 11 PM

File: helpers/addresses.py [hunk 1]
```diff
@@ -214,6 +214,7 @@
         "bFpMbtcHbtc": "0x26B8efa69603537AC8ab55768b6740b67664D518",
         "bbveCVX-CVX-f": "0x937B8E917d0F36eDEBBA8E459C5FB16F3b315551",
         "remBADGER": "0x6aF7377b5009d7d154F36FE9e235aE1DA27Aea22",
+        "remDIGG": "0x8B2a18b6400338272FDD1B991F5163E21723AF60",
```
Comment (petrovska-petro) [inline helpers/addresses.py:217]: @sajanrajdev scripts looks good and cli commands exec smoothly

just having a doubt on the sett address based on this [comment]( https://github.com/Badger-Finance/badger-strategies/issues/35#issuecomment-1026838777)

which is the right address 0x99F39D495C6A5237f43602f3Ab5F49786E46c9B0 VS 0x8B2a18b6400338272FDD1B991F5163E21723AF60?

Comment (sajanrajdev) [reply helpers/addresses.py:217]: Yes Petro, a new vault was redeployed this morning and I think Alex edited the comment there. Aas about to get to change it. Thanks for checking!

**Comment by sajanrajdev** [comment]
@petrovska-petro @tabshaikh Updated remDIGG address as per redeployment.

To wire up remDIGG call:
```
brownie run scripts/badger/wire_up_controller.py wire_up_controller 0x3F61344BA56df00dad9bBcA05d98CA2AeC43Ba0B 0x99F39D495C6A5237f43602f3Ab5F49786E46c9B0 0x4055D395361E73530D43c9D4F18b0668fe4B5b91
```

**Review by petrovska-petro — APPROVED** [review]
LGTM! ✅ 

Addresses verified against [comment](https://github.com/Badger-Finance/badger-strategies/issues/35#issuecomment-1026838777)

**Review by GalloDaSballo — APPROVED** [review]
Inputs are correct (as of latest)
Script will `approveStrategy` and then `setStrategy` lastly asserting the setup is correct.
LGTM

**Pull request merged**